src/targets/node/nvmResolver.ts

```diff
@@ -151,6 +164,15 @@ export class NvmResolver implements INvmResolver {
     return directory ? path.join(directory, 'bin') : undefined;
   }
 
+  private async resolveFnm(version: string, fnmHome: string) {
+    const directory = this.findBinFolderForVersion(
+      path.join(fnmHome, 'node-versions'),
+      `v${version}`,
+    );
+
+    return directory ? path.join(directory, 'installation', 'bin') : undefined;
```

Comment:
On Windows, there's no nested `bin` directory. `node.exe` is just in `path.join(directory, 'installation')`

Reply:
thanks for the review, solved it!